app/assets/stylesheets/_bootstrap_overrides.scss

```diff
@@ -4,14 +4,18 @@
   }
 }
 
-h1, h2, h5 {
-  font-weight: bold;
-}
-
 .modal-header {
   border-bottom: 0;
 }
 
+dd {
+  border-bottom: 1px solid $grey;
```

Comment:
This is pretty broad!! There are lots of `dd`s around - are you sure you want all of them to have a `border-bottom`?

You can also use a Bootstrap utility class to achieve this - add the class `border-bottom`

Reply:
Yup, I think at this stage yes...But might add to individual items in the future :smile: